schema/application/resolvers/ChecklistResult.ts

```diff
@@ -27,6 +27,22 @@ export const ChecklistResult: ChecklistResultResolvers = {
   auditable(parent, _, ctx) {
     return ctx.orm.auditable.load(parent.id);
   },
+  async order(parent) {
+    const { id } = decodeGlobalId(parent.id);
+    const [row] = await sql<
+      [
+        {
+          workresultoder: number;
```

Comment:
wasn't sure how to make this number or null

in the schema I have it as number or null, but was getting errors adding it here